dashboard/src2/pages/LoginSignup.vue

```diff
@@ -143,22 +194,42 @@
 									<span
 										class="relative bg-white px-2 text-sm font-medium leading-8 text-gray-800"
 									>
-										Or continue with
+										Or
 									</span>
 								</div>
 							</div>
-							<Button
-								:loading="$resources.googleLogin.loading"
-								@click="$resources.googleLogin.submit()"
-							>
-								<div class="flex items-center">
-									<GoogleIconSolid class="w-4" />
-									<span class="ml-2">Google</span>
-								</div>
-							</Button>
+
+							<div class="flex flex-col gap-2">
+								<!-- <Button
+									:route="{
+										name: 'Login',
+										query: { ...$route.query, use_otp: undefined },
+									}"
+								>
+									Continue with Password
+								</Button> -->
+								<Button
+									v-if="!useOTP"
+									:route="{
+										name: 'Login',
+										query: { ...$route.query, use_otp: 1 },
+									}"
+								>
+									Continue with Email Verification <Cc:ie></Cc:ie>ode
```

Comment:
👀

Reply:
Lol, idk how it got there (probably autocomplete). Removing.